### ISSUE TITLE
eRadio developers update

### ISSUE THREAD

**Issue by georgesofianosgr** [quassy/elementary-apps#9]
Please also add Fotini Skoti to authors.(https://github.com/fotinisk) I'm not the only one.

**Comment by quassy** [comment]
Thanks, I have added her. Is she also a co-developer of Dream Icon Installer? 

Feel free to report anything else you think is missing / wrong.

**Comment by georgesofianosgr** [comment]
Currently no. Generally all projects located here https://github.com/DreamDevel could be developed by me , fotini or both of us (and maybe by another future member of the team). Maybe is better to set author for our projects to "Dream Development" and point here (https://github.com/orgs/DreamDevel/people). But it's ok if you prefer to keep our names. 

Thanks for the change.

**Comment by quassy** [comment]
The [people link](https://github.com/orgs/DreamDevel/people) only gives me this: 

> This organization has no public members.

What is Feedo going to be? ;)

**Comment by georgesofianosgr** [comment]
Thanks for noticing we didn't know that we had privacy settings in the group. Now it should be fixed.

Feedo is a minimalist feedly app that we started back in September and we stopped development due to a Freya bug. As Feedler is in active development we are not sure if we are going to continue this project.

I would like also to point that the screenshot here (https://github.com/DreamDevel/eRadio/tree/clean_code) is more representative than the current screenshot. The initial eradio size is wrong due to the same bug prevented us from developing feedo.
